tensorflow/python/platform/sysconfig.py

```diff
@@ -80,7 +80,7 @@ def get_link_flags():
   if not _MONOLITHIC_BUILD:
     flags.append('-L%s' % get_lib())
     if is_mac:
-      flags.append('-l:libtensorflow_framework.%s.dylib' % ver)
+      flags.append('-ltensorflow_framework.%s' % ver)
     else:
       flags.append('-l:libtensorflow_framework.so.%s' % ver)
```

Comment:
Should we do a similar change on the `else:` branch?

Reply:
Thanks @mihaimaruseac. I think linux may not apply as linux 's .so is named as `libtensorflow_framework.so.1` (not `libtensorflow_framework.1.so` like on Mac) so we will need `:` (`-l:libtensorflow_framework.so.1`).